Samples/SimpleMvcExample.Messages/BackupDatabaseServer.cs

```diff
@@ -9,6 +9,7 @@ public class BackupDatabaseServer : ICommand
         public DatabaseServerId? DatabaseServerId { get; init; }
         
         [DisplayName("Destination Filename")]
+        [Description("Testing")]
```

Comment:
Is this supposed to be here?

Reply:
Yes, I needed a description to help style the Description of a field.